src/login/logind-user.h

```diff
@@ -34,11 +34,17 @@ struct User {
         char *state_file;
         char *runtime_path;
 
-        char *slice;                     /* user-UID.slice */
-        char *service;                   /* user@UID.service */
-        char *runtime_dir_service;       /* user-runtime-dir@UID.service */
+        /* user-UID.slice */
+        char *slice;
 
-        char *service_job;
+        /* user-runtime-dir@UID.service */
+        char *runtime_dir_service;
```

Comment:
maybe rename this to `runtime_dir_unit`? otherwise it's so weirdly assymetric that the `service_manager_unit` field below is called the way it is, i.e. unit vs. service...